site2/website-next/release-notes/versioned/pulsar-2.11.0.md

```diff
@@ -0,0 +1,484 @@
+- Fix MessageDeduplication#inactiveProducers may not be persistence correctly [#15206](https://github.com/apache/pulsar/pull/15206)
+- Cancel fencedTopicMonitoringTask when topic closed normally [#15202](https://github.com/apache/pulsar/pull/15202)
+- Fix npe when doCacheEviction [#15184](https://github.com/apache/pulsar/pull/15184)
+- Make health check fail if dead locked threads are detected [#15155](https://github.com/apache/pulsar/pull/15155)
+- Fix parameter saslJaasBrokerSectionName in broker.conf [#15110](https://github.com/apache/pulsar/pull/15110)
+- Fix potential to add duplicated consumer [#15051](https://github.com/apache/pulsar/pull/15051)
+- Fix rewind failed when redeliverUnacknowledgedMessages [#15046](https://github.com/apache/pulsar/pull/15046)
+- Fix race condition in updating lastMarkDeleteEntry field [#15031](https://github.com/apache/pulsar/pull/15031)
+- Avoid heartbeat topic to offload [#15008](https://github.com/apache/pulsar/pull/15008)
+- Evicting cache data by the slowest markDeletedPosition [#14985](https://github.com/apache/pulsar/pull/14985)
+- Return if reset in progress [#14978](https://github.com/apache/pulsar/pull/14978)
+- Dynamic update broker-level subscribe-rate limter [#14890](https://github.com/apache/pulsar/pull/14890)
+- Fix wrong state for non-durable cursor [#14869](https://github.com/apache/pulsar/pull/14869)
+- Support advertised listeners for HTTP and HTTPS services [#14839](https://github.com/apache/pulsar/pull/14839)
+- Support dynamic update between non-zero values of topicPublisherThrottlingTickTimeMillis [#14782](https://github.com/apache/pulsar/pull/14782)
+- Support dynamic update between non-zero values of brokerPublisherThrottlingTickTimeMillis [#14759](https://github.com/apache/pulsar/pull/14759)
+- Fix incorrect entryId in warn log when reading entry from tiered storage [#14685](https://github.com/apache/pulsar/pull/14685)
+- Optimize PositionImpl toString, compareTo and hashCode methods [#14678](https://github.com/apache/pulsar/pull/14678)
+- Improve readability of ManagedCursorImpl source code [#14617](https://github.com/apache/pulsar/pull/14617)
+- Intercept ACK command by BrokerInterceptor [#14616](https://github.com/apache/pulsar/pull/14616)
+- Add filteredMsgCount for pulsar-admin broker-stats topics [#14531](https://github.com/apache/pulsar/pull/14531)
+- Fix NPE when subscription is already removed [#14363](https://github.com/apache/pulsar/pull/14363)
+- Fix backpressure handling in Jetty web server configuration [#14353](https://github.com/apache/pulsar/pull/14353)
+- Optimize find nics process [#14340](https://github.com/apache/pulsar/pull/14340)
+- Support pass http auth status [#14044](https://github.com/apache/pulsar/pull/14044)
+- Support caching to drain backlog consumers [#12258](https://github.com/apache/pulsar/pull/12258)
+- Strict bookie affinity group strategy [#12025](https://github.com/apache/pulsar/pull/12025)
+
+### Clients
```

Comment:
Is this part (line 217-285) already split into the following independent release notes? If so, should this part be removed here?

<img width="206" alt="image" src="https://user-images.githubusercontent.com/50226895/185337812-c1721ed6-dfd5-4915-971a-3aa886e31ac0.png">